src/librustc_session/options.rs

```diff
@@ -814,12 +811,8 @@ options! {DebuggingOptions, DebuggingSetter, basic_debugging_options,
         "for every macro invocation, print its name and arguments"),
     debug_macros: bool = (false, parse_bool, [TRACKED],
         "emit line numbers debug info inside macros"),
-    generate_arange_section: bool = (true, parse_bool, [TRACKED],
-        "generate DWARF address ranges for faster lookups"),
```

Comment:
Why is this removed?

Reply:
Oh, I read the comment on `parse_bool` now. It seems like the behavior of `parse_bool` is not intentional. I'm OK with removing the kind of outdated `-Z incremental-queries` flags, but for `-Z generate-arange-section` it seems useful to enable opting out.

Reply:
So that will require either changing it to `-Z no-generate-arange-section`. Is that ok?

Reply:
I have made this change, and split it into a separate commit.